app/src/main/java/com/li/weatherapp/ui/news/NewsActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.li.weatherapp.ui.news
+
+import android.content.Context
+import android.content.Intent
+import android.webkit.WebViewClient
+import com.li.weatherapp.R
+import com.li.weatherapp.base.BaseActivity
+import kotlinx.android.synthetic.main.activity_news.*
+
+class NewsActivity : BaseActivity() {
+
+    override val layoutResource: Int = R.layout.activity_news
```

Comment:
```suggestion
    override val layoutResource: Int get()= R.layout.activity_news
```
2 cách này khác nhau 1 vài chút đấy, em thử tìm hiểu thêm xem sao 
suggest: xem Kotlin byte code

Reply:
vâng

Reply:
nếu là Int = R.layout.activity_news thì nó tạo biến layoutResource rồi getField từ biến đó trong getLayoutResource(), còn get() thì load constant luôn trong getLayoutResource()